src/sage/plot/animate.py

```diff
@@ -74,9 +83,13 @@
 
     sage: (x,y,z) = SR.var("x,y,z")
     sage: def frame(t):
-    ....:     return implicit_plot3d((x^2 + y^2 + z^2), (x, -2, 2), (y, -2, 2), (z, -2, 2), plot_points=60, contour=[1,3,5], region=lambda x,y,z: x<=t or y>=t or z<=t)
-    sage: a = animate([frame(t) for t in srange(.01,1.5,.2)])
-    sage: a[0]       # long time
+    ....:     return implicit_plot3d((x^2 + y^2 + z^2),
+    ....:                            (x, -2, 2), (y, -2, 2), (z, -2, 2),
+    ....:                            plot_points=60, contour=[1,3,5],
+    ....:                            region=lambda x,y,z: x<=t or y>=t or z<=t)
+    sage: a = animate([frame(t)
+    ....:              for t in srange(.01,1.5,.2)])
+    sage: a[0]                      # long time
     Graphics3d Object
     sage: a.show()                  # long time  # optional -- ImageMagick
```

Comment:
Just to be sure, are both forms accepted ? `# optional -- ImageMagick` and `# optional - ImageMagick`

Reply:
All of these forms accepted; `sage -fixdoctests` normalizes to the `# optional - imagemagick` form.